docs/features/roll-editor.md

```diff
@@ -30,10 +30,64 @@ The metadata for a weapon will show what damage type it is, is it a primary, sec
+
+This part of the document seeks to answer most questions about various stats, however, it should be noted that Destiny 2 is a very complex game. More in-depth explainations of these stats and scenarios will be expanded upon in their own document. 
+
 ### Basic
 
+The basic stats are there to give you a rough idea of how a weapon feels and operates. They should only be compared with weapons of the same type as stats such as impact are relative to the base impact number of a weapon type. The aim of the basic stat section is to describe the stats in relation to other weapons of the same type as well as give small examples as to how they work in game.
+
+#### Impact
+
+Impact is the damage stat on most weapons. This number can be a little misleading, however, as it can only be used to reference damage within the same weapon type. E.g. the hand cannon "Targeted Redaction" has an impact stat value of 92 and does 80 crit damage in the crucible. Compare that to the combat bow "Tyranny of Heaven" which has an impact stat of 68 and does 137 crit damage in the crucible. For the most part, impact values are the same across a given frame for a weapon type. It mainly comes into play with fusion rifles, including linear fusions, and swords. Those weapon types can augement their impact stat via certain perk selections.
+
+If the weapon family is the same and the impact value is higher then impact translates to the difference between the two numbers as the difference in damage percentage. The "Half Truths" and "Bequest" are both arc, adaptive frame swords. The "Bequest" has an impact of 70 and the "Half Truths" has an impact of 60. This translates to the "Bequest" doing 10% more damage per swing than the "Half Truths". 
+
+#### Range
+
+The range stat of a weapon seeks to show how far a weapon can do max damage before it experiences a damage dropoff. This stat alone is what defines the hipfire range of a weapon's type. In order to find a weapon's ADS damage falloff you need to take the hipfire fall of distance in meters and multiply it by the weapon's zoon divided by ten. E.g. the "Autumn Wind" has a hipfire damage falloff of 18.69m and its zoom is 17. So if you multiply 18.69 * (17/10) you will get 31.773m. This is slightly different than the value that you will see listed on a weapon's page. Our exact formula accounts for the weapon moving backwards when you ADS, thus reducing the zoom value by .25. More details on this formula can be found in our [range explaination document](../calculations/range.md).
```

Comment:
A few typos here -- "weapon's _zoon_", and "range _explaination_", "fallof"

Also not to be super annoying about style/consistency but I think the my Cliff Notes of a style guide is:
- Capitalize Bungie main stat names: "A weapon's Zoom, Recoil Direction"
- Additionally, capitalize perk/trait/frame names, weapon names, and weapon families. They don't need quotes or a definite article: "Quickdraw rolls on The Palindrome (Adept). It's my favorite Adaptive Frame Hand Cannon. I use it with Lucky Pants."
- Our calculated/derived stats can be lowercased: "flinch resistance"
- Prefer using numbers for any sort of stat or value in an equation rather than text: "10" not "ten"
- For equations, try to use our TeX formatting since it'll be more readable as math
- "hip-fire" not "hipfire"
- "damage falloff distance" not "damage dropoff" -- just "damage falloff" can be okay as shorthand
  - If the above applies only to start or end, it should make mention. "damage falloff start distance"

That being said, I'm totally cool to do the proofreading myself after this is merged rather than playing PR phone tag 😅 Still kind of formulating my ideas on communicating stuff

Reply:
oh my I thought I put everything through a word processor for spell check ahhh that's a little embarrassing!! I can do these proofreading changes and in fact I really appreciate that you wrote these guidelines as I genuinely felt a little bad by just assuming certain styles haha. This will definitely help in the future (borderline might be a good idea to have in the repo somewhere for others)

Reply:
also for whatever its worth I don't mind discord DMs or if you want to create a PR review channel in the server that could be faster than PR comments